ios/Compass/AppDelegate.m

```diff
@@ -22,9 +19,8 @@ @implementation AppDelegate
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
   if ([FIRApp defaultApp] == nil) {
-  //   [FIRApp configure];
+     [FIRApp configure];
```

Comment:
I just forgot to re-add the comment, will fix it.

But apart from that it now finally works for you?

Reply:
Nevermind, i just read your other comment, stating that the missing config file for fcm caused the immediate crash

Reply:
Yep - works now.

The first problem is/was my usage of [NVM](https://github.com/nvm-sh/nvm). React-Native comes with a script to identify the currently used node version (located at `node_modules/react-native/scripts/find-node.sh).` Till RN 0.63.4 this works like a charm but with 0.64 a change occurred that can potentially hinder RN to get the right Node version when using NVM in combination with multiple aliases (seems to be still an issue). This can be fixed by running:

`nvm unalias default`

As NVM is widely used, this might by worth documenting. The second issue was just the uncommented line in AppDelegate.m.